src/main/java/io/github/Alathra/AlathranCommands/commands/CommandWildTp.java

```diff
@@ -44,9 +55,15 @@ public CommandWildTp (AlathranCommands pl) {
                     throw CommandAPIBukkit.failWithAdventureComponent(ColorParser.of(TPCfg.get().getString("Messages.error-cooldown")).parseMinimessagePlaceholder("cooldown", CooldownManager.getInstance().getRemainingCooldownString(p, CooldownType.TELEPORT_WILDERNESS)).build());
                 }
 
+                if (teleportCurrentWorld) {
+                    teleportWorld = p.getWorld();
+                } else {
+                    teleportWorld = Bukkit.getWorld(TPCfg.get().getString("Settings.WildTP.Destination-world"));
+                }
+
                 final long grace = Duration.ofSeconds(TPCfg.get().getInt("Settings.WildTP.Grace.Time")).toMillis();
 
```

Comment:
Null check `teleportWorld` in case the config entry defines a non-existing dimension or said dimension becomes unavailable during runtime.